governance/api-reviews.md

```diff
@@ -19,7 +19,20 @@ the comments to get an answer.
 
 ## High level points
 
+### Python Versions
+
+TensorFlow supports a range of Python versions, and changes need to be
+compatible with all of them. This means that language features not available in
+TensorFlow's minimum supported version cannot be used. 
+
+We regularly reconsider the range of supported versions based on the number of
```

Comment:
Also consider end of life timelines? https://endoflife.date/python

Reply:
As seen by Python2, these are mostly aspirational. Realistically, we will make these decisions based on users, not timelines.

Reply:
Are we suggesting to contributors to develop with the minimal supported python version as in `https://www.tensorflow.org/install`?

Cause `docker run --rm tensorflow/tensorflow:devel python3 -V` is `Python 3.6.9`

Reply:
OSS TF presubmit tests are targeted to Python 3.6. But since we do have tags like `no_oss_py35` etc., it suggests that TF team might run those tests internally for other Python versions as well.